xscen/scripting.py

```diff
@@ -22,6 +22,8 @@
 
 logger = logging.getLogger(__name__)
 
+__all__ = []
```

Comment:
I can't really add functions here in Git (it won't let me make the indents correctly...). The following functions should be in there: `send_mail, send_mail_on_exit, measure_time, timeout, TimeoutException`